### PR TITLE
fix: simple job control button

### PR DESCRIPTION
**Pull request by collinlokken**
## What does this pull request change?
- Turns the job controll button (run/stop/restart) into a simple button with only one icon
- Do stop and create in two steps

### CODE REVIEW

**Comment by collinlokken** [comment]
image (1)
image (2)
image (3)
image (4)
image

File: packages/dm-core-plugins/src/job/JobControlButton.tsx [hunk 1]
```diff
@@ -13,57 +13,70 @@ export const JobControlButton = (props: {
 }) => {
   const { jobStatus, createJob, asCronJob, disabled, exists, remove } = props
   const [hovering, setHovering] = useState(false)
+  const [buttonColor, setButtonColor] = useState<
+    'primary' | 'secondary' | 'danger' | undefined
```
Comment (henrikbossart) [inline packages/dm-core-plugins/src/job/JobControlButton.tsx:17]: Why would the color ever be undefined? 🤔

Comment (collinlokken) [reply packages/dm-core-plugins/src/job/JobControlButton.tsx]: Does not really make sense, removed it 👍

**Review by henrikbossart — APPROVED** [review]
The red danger color is _very_ red, but I guess its ok... 🤷‍♂️

**Comment by soofstad** [comment]
Looks good!
The red color should be the same red color.

**Comment by henrikbossart** [comment]
@soofstad I think the red is the same, just in hover state and not

**Pull request merged**